### PR TITLE
RoyalApps.RoyalTS version 6.1.60512.0

### PR DESCRIPTION
**Pull request by StefanKoell**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106655)

### CODE REVIEW

**Comment by StefanKoell** [comment]
Can someone check the hash mismatch policy? I double checked the SHA 256 hash of the MSI and it is correct.

**Comment by rodrigoborgesdeoliveira** [comment]
If you're still seeing the other hash on your end, it could be that different regions of the world are getting served with a different msi file.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> ##[error] Installer https://download.royalapps.com/RoyalTS/RoyalTSInstaller_6.01.60512.0.msi 
> ##[error] Result: Failed 
> ##vso[task.logissue type=error] Specified hash doesn't match. 
> Actual hash:   26884BBD515F4701C070BA18C0518B64055C387113D9019E4BCFD89CE8A43A16 
> Expected hash: 58e7a7dc5eb12c3bb337de2554e0dd35f3504514715114b27d7b15b1c97bbd94

File: manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.1.60512.0
+MinimumOSVersion: 10.0.0.0
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download.royalapps.com/RoyalTS/RoyalTSInstaller_6.01.60512.0.msi
+  InstallerSha256: 58E7A7DC5EB12C3BB337DE2554E0DD35F3504514715114B27D7B15B1C97BBD94
```
Comment (stephengillie) [inline manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.installer.yaml:11]:
```suggestion
  InstallerSha256: 26884BBD515F4701C070BA18C0518B64055C387113D9019E4BCFD89CE8A43A16
```

**Comment by StefanKoell** [comment]
I think I know what was going on. I do use cloudflare services and it seems an outdated version was cached >>somewhere<<. Putting a random querystring at the end to the download URL downloaded the correct version with the correct/proposed hash mentioned here.

I'm sorry for the inconvenience. Thanks for the proposed file change @stephengillie

File: manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.1.60512.0
+PackageLocale: en-US
+Publisher: Royal Apps GmbH
+PackageName: Royal TS
+PackageUrl: https://www.royalapps.com/ts/win/features
+License: Copyright 2021 Royal Apps GmbH
```
Comment (mdanish-kh) [inline manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.locale.en-US.yaml:10]:
```suggestion
License: Proprietary
```
A copyright isn't a license

File: manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.1.60512.0
+MinimumOSVersion: 10.0.0.0
```
Comment (mdanish-kh) [inline manifests/r/RoyalApps/RoyalTS/6.1.60512.0/RoyalApps.RoyalTS.installer.yaml:6]:
```suggestion
MinimumOSVersion: 10.0.0.0
Scope: machine
```
Package installs machine-wide (for all users)

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by StefanKoell** [comment]
@mdanish-kh thanks for the additional commit suggestions. I'm wondering if the next time I run wingetcreate.exe update ... these changes are automatically pushed forward? Or do I have to do something special in order to get it right next time?

**Comment by mdanish-kh** [comment]
> @mdanish-kh thanks for the additional commit suggestions. I'm wondering if the next time I run wingetcreate.exe update ... these changes are automatically pushed forward? Or do I have to do something special in order to get it right next time?

`wingetcreate update` will always fetch the latest manifest from the repo and use that as the base manifest for the update. These changes will carry forward the next time you run `wingetcreate update` provided that the manifest hasn't been modified in the repo.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**